ui-admin/src/portal/publish/diffComponents.test.tsx

```diff
@@ -1,4 +1,4 @@
-import { render, screen } from '@testing-library/react'
+import { fireEvent, render, screen } from '@testing-library/react'
```

Comment:
Generally, `@testing-library/user-event` is preferred over `fireEvent`.
https://testing-library.com/docs/user-event/intro/#differences-from-fireevent

There's some places I've used `fireEvent.change` because it was more convenient when testing controlled components vs adding a test wrapper that handled state updates.